src/Misc/MiddleWare.cpp

```diff
@@ -714,6 +718,13 @@ class MiddleWareImpl
     // Handle an event with special cases
     void handleMsg(const char *msg);
 
+    // Add a message for handleMsg to a queue
+    void queueMsg(const char* msg)
+    {
+        msgsToHandle.emplace(msg, msg+rtosc_message_length(msg,
+                                          std::numeric_limits<size_t>::max()));
```

Comment:
If I'm not mistaken rtosc_message_length(msg, -1) also does this same behavior.

Reply:
`-1` is also used in other places, so I think it would look more consistent.